libc/newhdrgen/yaml/sys/mman.yaml

```
@@ -98,6 +98,16 @@ functions:
       - type: void *
       - type: size_t
       - type: int
+  - name: remap_file_pages
+    standards:
+      - LINUX
```

nit: in other places it's capitalized as `Linux`
```suggestion
      - Linux
```